packages/components/src/Illustration/Scene/BrandMomentCaptureIntro/_docs/BrandMomentCaptureIntro.stickersheet.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import { Meta } from "@storybook/react"
+import {
+  StickerSheet,
+  StickerSheetStory,
+} from "~storybook/components/StickerSheet"
+import { BrandMomentCaptureIntro } from "../index"
+
+export default {
+  title: "Components/Illustrations/BrandMomentCaptureIntro",
+  parameters: {
+    chromatic: { disable: false },
+    controls: { disable: true },
+  },
+} satisfies Meta
+
+const StickerSheetTemplate: StickerSheetStory = {
+  render: ({ isReversed }) => (
+    <StickerSheet isReversed={isReversed}>
+      <StickerSheet.Header headings={["Default", "Autoplay (hover)"]} />
```

Comment:
Looks like there's only one example, so no need for headings right?

Reply:
Ah fudge, must of unstaged one, will fix!